src/js/events/ChronosActions.js

```diff
@@ -46,18 +55,126 @@ const ChronosActions = {
     },
     {delayAfterCount: Config.delayAfterErrorCount}
   ),
+
+  fetchJobDetail: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
+      data: [
+        {name: 'embed', value: 'activeRuns'},
+        {name: 'embed', value: 'schedules'}
+      ],
+      success: function (response) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_SUCCESS,
+          data: response,
+          jobID
+        });
+      },
+      error: function (xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_ERROR,
+          data: RequestUtil.parseResponseBody(xhr),
+          jobID,
+          xhr
+        });
+      },
+      hangingRequestCallback: function () {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_ONGOING,
+          jobID
+        });
+      }
+    });
+  },
+
+  deleteJob: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
+      data: {stopCurrentJobRuns: true},
```

Comment:
This needs to be part of the query string and, as far as I know, the `RequestUtil` will only append data as query parameter if you perform `GET` request. Please append this manually and use `false` as a default.
If a user tries to delete a job with active runs, we will get an error and should display a confirm dialog asking the user if she want's to force the deletion and stop all current runs.